### PR TITLE
Switch to NPF

### PR DESCRIPTION
**Pull request by tomrf1**
## Why are you doing this?
The NPF did well, we're moving all users over.

Note - there is a lot of cleanup to be done later

[**Trello Card**](https://trello.com)

## Changes

* Change 1
* Change 2

### CODE REVIEW

File: app/controllers/SiteMap.scala [hunk 1]
```diff
@@ -21,6 +21,7 @@ class SiteMap(
     Ok(sitemapResponse)
   }
 
+  // TODO: more cleanup
```
Comment (joelochlann) [inline app/controllers/SiteMap.scala]: I added this comment but this is actually orthogonal to all of this stuff so you can delete it

**Comment by tomrf1** [comment]
I'll merge this on Monday
